sensing/imu_corrector/config/gyro_bias_estimator.param.yaml

```diff
@@ -1,6 +1,6 @@
 /**:
   ros__parameters:
-    gyro_bias_threshold: 0.0015 # [rad/s]
-    velocity_threshold: 0.0 # [m/s]
-    timestamp_threshold: 0.1 # [s]
+    gyro_bias_threshold: 0.0050 # [rad/s]
```

Comment:
This value was chosen so that the OK/NG decision is appropriately split for the two internal data.
However, the results of the survey revealed that the original value should remain the same, so it was reverted.
https://github.com/autowarefoundation/autoware.universe/pull/5259/commits/b012405bbe8d1104798e71641a395af8384a26cc